brownie/data/network-config.yaml

```diff
@@ -38,6 +38,18 @@ live:
         id: kotti
         host: https://www.ethercluster.com/kotti
         explorer: https://blockscout.com/etc/kotti/api
+  - name: Binance Smart Chain
+    networks:
+      - name: bsc-test
+        chainid: 97
+        id: bsc-test
+        host: https://data-seed-prebsc-1-s1.binance.org:8545
+        explorer: https://api-testnet.bscscan.com/api
+      - name: bsc-main
```

Comment:
A more descriptive name than just `bsc-main` would be good.  Same for the testnet.